assignments/assignment01.rb

```diff
@@ -67,8 +103,33 @@ def my_reverse(s)
 
 # Write a method `palindrome?`
 # that determines whether a string is a palindrome
+  
 def palindrome?(s)
-  # your implementation here
+  
+  # => PSEUDO CODE
+  # => -----------------------------------------------------------------
+  # => count the length of the string use it as the countdown
+  # => iterate string to the last character - zero index
+  # => compare last char and first char if the same then go to the nex if not the exit
+  # => return true of false
+  # => ------------------------------------------------------------------
+  
+  firstChar = 0
+  lastChar = s.length - 1
+  while firstChar < s.length
+    s = s.downcase
+    if s[firstChar].eql?(s[lastChar])
+      print true
+      return true
```

Comment:
you'll exit the loop on the first test, even though you have lots of characters left to test